.travis.yml

```diff
@@ -3,7 +3,9 @@ os:
     - linux
     - osx
 julia:
-    - 0.6
+    - 0.7
+    - 1.0
+    - nightly
```

Comment:
Nit: We've been omitting `nightly` from JuliaOpt travis builds because of long CI waits. Not really important if we don't expect CI to run often on Convex.jl

Reply:
I wouldn't necessarily expect it to run often but I'm happy to remove nightly if you think that's best.